deploy/infra/main.bicep

```diff
@@ -32,6 +32,9 @@ param orchestrationModulePrefix string = 'orc'
 @description('Specify whether or not to deploy batch account')
 param deployBatchAccount bool = true
 
+@description('Specify whether or not to deploy PostgreSQL')
+param deployPgSQL bool = true
```

Comment:
@mandarinamdar Same here. I would say this feature should be off by default unless enabled. Thoughts?

Reply:
I vote default to true until we have logic to disable or remove postgres step from the data pipeline. We don't want to break regression tests.